test/uk/gov/hmrc/traderservices/journey/TraderServicesFrontendFormatSpec.scala

```diff
@@ -90,12 +90,12 @@ class TraderServicesFrontendFormatSpec extends UnitSpec {
         )
       )
       validateJsonFormat(
-        """{"state":"AnswerExportQuestionsFreightType","properties":{"model":{"declarationDetails":{"epu":"123","entryNumber":"Z00000Z","entryDate":"2020-10-05"},"exportQuestionsAnswers":{"requestType":"Hold","priorityGoods":"LiveAnimals"}}}}""",
+        """{"state":"AnswerExportQuestionsFreightType","properties":{"model":{"declarationDetails":{"epu":"123","entryNumber":"Z00000Z","entryDate":"2020-10-05"},"exportQuestionsAnswers":{"requestType":"C1601","priorityGoods":"LiveAnimals"}}}}""",
```

Comment:
Most of this is fine, the only thing is now that users won't be bypassing the route page anymore they'll always have to answer it - some of the tests have a step to check but not all do - some will need to be added

ie. 
<img width="505" alt="image" src="https://user-images.githubusercontent.com/55190161/98260037-bf3f4d00-1f7a-11eb-8713-acd3aea861c3.png">